Ticket review (#527)
foundriesio/docs: source/getting-started/install-fioctl/index.rst

     Client secret:
     You are now logged in to Foundries.io services.

.. seealso::
   :ref:`ref-fioctl` documentation.

.. _Github Releases: https://github.com/foundriesio/fioctl/releases

.. _gs-git-config:

Configuring Git
###############

> Pushing to your repositories with :ref:`FoundriesFactory® <ref-factory-definition>` is a matter of configuring Git to use :ref:`Fioctl™ <ref-fioctl>` as a credential helper.

Reviewer: This "is a matter of" is over when you think about wordlessness

> Afterwards, Git will know when you are connecting to ``source.foundries.io`` and will use :ref:`ref-fioctl` for authentication when utilizing ``git`` commands.

Reviewer: ```suggestion
Afterward, Git will know when you connect to ``source.foundries.io`` and will use :ref:`ref-fioctl` for authentication when utilizing ``git`` commands.
```

Setting Up Git
^^^^^^^^^^^^^^

Run the following command to add the relevant entries to the Git configuration:

.. prompt:: bash host:~$, auto

   host:~$ sudo fioctl configure-git

.. important::

>    This needs to be run as ``sudo`` instead of directly as the ``root`` user.

Reviewer: ```suggestion
   This must be run as ``sudo`` instead of directly as the ``root`` user.
```

   This is because it needs to have privileges to create a symlink in the same directory as where ``git`` is located.

.. warning::
   * Existing users reconfiguring Git access may need to remove the following lines from ``.gitconfig`` to use ``fioctl configure-git`` utility::

      [http "https://source.foundries.io"]
      extraheader = Authorization: basic <TOKEN>

   * If editing scopes on existing tokens, the user should refresh the local ``fioctl`` credentials with::

      fioctl login --refresh-access-token

> Verify that this has been successful by cloning a repository from your Factory,
> such as your ``containers.git`` repo.

Reviewer: ```suggestion
Verify this has succeeded by cloning a repository from your Factory,
such as your ``containers.git`` repo.
```
